std/testing/asserts_test.ts

```diff
@@ -159,6 +160,19 @@ Deno.test("testingNotEquals", function (): void {
   assertEquals(didThrow, true);
 });
 
+Deno.test("testingAssertExists", function (): void {
+  assertExists("Denosaurus");
+  let didThrow;
+  try {
+    assertExists(undefined);
```

Comment:
We should test `null` and `void 0` in the same way.

Reply:
@kitsonk I added some falsy test